app/src/lib/stores/app-store.ts

```diff
@@ -2439,6 +2439,9 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
         if (fetchType === FetchType.UserInitiatedTask) {
```

Comment:
Why are we only doing this for manual fetches? Shouldn't we be refreshing PRs and issues in the background?

Reply:
We avoid doing this for background fetching because we already have a `PullRequestUpdater` that's working in the background and updating every 10mins - #3502 has some more context for why we went down that path. I think the goal here is to force a refresh of PRs as a part of the user fetching changes from the remote repository.

We don't really have an equivalent for fetching issues in the background currently - `IssuesAutocompletionProvider` manages this inside it's own `ThrottledScheduler` which runs ever minute, but this is only started _after_ an autocompletion request is initiated - and the list won't be refreshed when it does have new results (I think you'd need to change the input to re-render the list).

https://github.com/desktop/desktop/blob/d73112a9fc9ca4a953697ae471d3942ae8e07d7b/app/src/ui/autocompletion/issues-autocompletion-provider.tsx#L51-L59